src/overlap_check/ProgressBar.cpp

```diff
@@ -62,7 +60,9 @@ void ProgressBar::set_value(double val) {
 
   // write the bar to screen
   std::cout << '\r' << bar.str() << std::flush;
+  need_final_newline = true;
```

Comment:
Can we make this default value `true` and only change it when needed in the block below?

Reply:
Isn't that kind of what's happening here?  Doesn't it need to be set to `true` each time `set_value` is called?

Reply:
Sorry that wasn't entirely clear. I meant the default value specified in `ProgressBar`'s declaration in the header file.